### PR TITLE
[12.0] product_supplierinfo_barcode and stock_barcodes_supplierinfo

### PR DESCRIPTION
**Pull request by eLBati**
Process products searching them by supplier barcode

### CODE REVIEW

**Review by mrcast — APPROVED** [review]
Functional test ok!
Thanks

File: stock_barcodes_supplierinfo/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "category": "Extra Tools",
+    "website": "https://github.com/OCA/stock-logistics-barcode",
+    "author": "TAKOBI, Odoo Community Association (OCA)",
+    "maintainers": ["eLBati"],
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "preloadable": True,
+    "depends": [
+        "product_supplierinfo_barcode",
```
Comment (legalsylvain) [inline stock_barcodes_supplierinfo/__manifest__.py:18]: Hi. This one should be also auto_install to True, don't you think ?

**Review by legalsylvain — CHANGES_REQUESTED** [review]
A single remark. LGTM otherwise. Thanks. (code review / no test).

File: stock_barcodes_supplierinfo/__manifest__.py (hunk 1 above)
Comment (eLBati) [reply stock_barcodes_supplierinfo/__manifest__.py:18]: Done, thanks

**Review by legalsylvain — APPROVED** [review]
Could uou check Travis ?

**Comment by eLBati** [comment]
@legalsylvain thanks, fixed travis

**Pull request merged**